tsconfig.build.json

```diff
@@ -1,21 +1,34 @@
 {
   "extends": "./tsconfig.json",
-  "angularCompilerOptions": {
-    "flatModuleId": "AUTOGENERATED",
-    "flatModuleOutFile": "AUTOGENERATED"
-  },
   "compilerOptions": {
-    "paths": {
-      "@ngxs/*": ["@ngxs/*"]
-    },
+    "outDir": "out-tsc/lib",
     "target": "es2015",
     "module": "es2015",
     "moduleResolution": "node",
     "declaration": true,
     "sourceMap": true,
     "inlineSources": true,
+    "importHelpers": true,
     "skipLibCheck": true,
-    "importHelpers": true
+    "types": [],
+    "lib": [
+      "dom",
+      "es2017"
+    ],
+    "paths": {
+      "@ngxs/*": ["@ngxs/*"]
+    },
+  },
+  "angularCompilerOptions": {
+    "annotateForClosureCompiler": true,
```

Comment:
What are the advantages?

Reply:
this will generate some information in output, which could be used by user using closure.

Reply:
Does it increase the bundle?

Reply:
good